jstz_cli/src/accounts/mod.rs

```diff
@@ -0,0 +1,54 @@
+use anyhow::Result;
+use bip39::{Language, Mnemonic, MnemonicType};
+use clap::Subcommand;
+
+pub mod account;
+
+use crate::config::Config;
+
+fn generate_passphrase() -> String {
+    let mnemonic = Mnemonic::new(MnemonicType::Words12, Language::English);
+    return mnemonic.to_string();
+}
+
+fn create_account(
+    passphrase: Option<String>,
+    alias: String,
+    cfg: &mut Config,
+) -> Result<()> {
+    let passphrase = match passphrase {
+        Some(passphrase) => passphrase,
+        None => {
+            let passphrase = generate_passphrase();
+            println!("Generated passphrase: {}", passphrase);
+            passphrase
+        }
+    };
+
+    let account = account::Account::from_passphrase(passphrase, alias)?;
+    cfg.accounts().upsert(account.clone());
+    cfg.save()?;
+
+    println!("Account created with address: {}", account.get_address());
```

Comment:
Unnecessary `.clone`. I recommend lifting the `println` then modifying the config